sample/prebid-config.yaml

```diff
@@ -19,6 +19,6 @@ settings:
 gdpr:
   vendorlist:
     filesystem-cache-dir: /var/tmp
-  geolocation:
-    enabled: false
+geolocation:
```

Comment:
Let's not turn off geolocation in this sample configuration as its purpose is illustrating minimal working configuration for which enabled geolocation doesn't come in the way.

Reply:
That is why this property is here for - `enabled: false`. Or did you mean to remove it completely?

Reply:
It would be better to remove it altogether as it just clutters this sample configuration.